### PR TITLE
feat(kuma-cp) sync policies over KDS

### PR DESCRIPTION
**Pull request by lobkovilya**
### Summary

Syncing policies over KDS

### Issues resolved

Fix https://github.com/Kong/kuma/issues/804

### Documentation

- [ ] Link to the website [documentation PR](https://github.com/Kong/kuma-website/pull/XXX)

### CODE REVIEW

File: app/kumactl/data/install/k8s/control-plane/kuma-cp/app.yaml [hunk 1]
```diff
@@ -33,7 +33,6 @@ spec:
 {{- end }}
   selector:
     app: kuma-control-plane
-{{- if eq .KumaCpMode "remote" }}
 ---
```
Comment (nickolaev) [inline app/kumactl/data/install/k8s/control-plane/kuma-cp/app.yaml:36]: global-remote-sync is not needed in Standalone mode

Comment (nickolaev) [reply app/kumactl/data/install/k8s/control-plane/kuma-cp/app.yaml:36]: I think you also miss 5685 as a containerPort later in this file.

File: pkg/plugins/runtime/k8s/plugin.go [hunk 2]
```diff
@@ -41,9 +41,9 @@ func init() {
 }
 
 func (p *plugin) Customize(rt core_runtime.Runtime) error {
-	if rt.Config().Mode == core_config.Global {
-		return nil
-	}
+	//if rt.Config().Mode == core_config.Remote {
+	//	return nil
+	//}
```
Comment (nickolaev) [inline pkg/plugins/runtime/k8s/plugin.go:46]: Hmm? just remove it if we don't need it

Comment (lobkovilya) [reply pkg/plugins/runtime/k8s/plugin.go:46]: Yep, thanks, I didn't notice

File: test/framework/constants.go [hunk 3]
```diff
@@ -14,6 +14,8 @@ const (
 
 	DefaultRetries = 30
 	DefaultTimeout = 3 * time.Second
+
+	LocalCPSyncNodePort = 30681
```
Comment (nickolaev) [inline test/framework/constants.go:18]: Should we change this one to 30685 (also in app.yaml) for consistency?

File: pkg/plugins/runtime/k8s/plugin.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (lobkovilya) [reply pkg/plugins/runtime/k8s/plugin.go:46]: Thanks, fixed

**Pull request merged**